ui/src/buckets/components/BucketRow.tsx

```diff
@@ -38,6 +38,7 @@ interface Props {
 class BucketRow extends PureComponent<Props & WithRouterProps> {
   public render() {
     const {bucket, onDeleteBucket} = this.props
+
```

Comment:
this doesn't look like a needed change :)

Reply:
I noticed that too